### DISCUSSION TITLE
Minimal docker image for actix 3

### DISCUSSION THREAD

**Discussion by serhii-tokariev84** [actix/actix-web#2557] [General] [answered]
Hi, could anybody share the dockerfile appropriate to run actix 3? For now I use rust docker image (https://hub.docker.com/_/rust) but that image is huge and pretty slow to start.

**Comment by robjtede** [comment, marked as answer]
This is the best we've got on hand: https://github.com/actix/examples/tree/master/basics/docker_sample

**Reply by serhii-tokariev84** [reply]
Thanks a lot
